src/components.d.ts

```diff
@@ -5,77 +5,67 @@
  */
 
 
-import '@stencil/core';
-
-
+import { HTMLStencilElement, JSXBase } from '@stencil/core/internal';
 
 
 export namespace Components {
-
-  interface StopWatchBox {
-    'applyHover': boolean;
-  }
-  interface StopWatchBoxAttributes extends StencilHTMLAttributes {
-    'applyHover'?: boolean;
-  }
-
   interface StopWatch {
     'applyHover': boolean;
     'hours': string;
     'milliseconds': string;
     'minutes': string;
     'seconds': string;
   }
-  interface StopWatchAttributes extends StencilHTMLAttributes {
-    'applyHover'?: boolean;
-    'hours'?: string;
-    'milliseconds'?: string;
-    'minutes'?: string;
-    'seconds'?: string;
+  interface StopWatchBox {
+    'applyHover': boolean;
```

Comment:
@peterennis is there a particular reason why we're making this property mandatory?

Reply:
Hover is part of the original code. The demo shows one with hover and one without. If you want to change it then you could have a prop value. Not part of this PR.